man/systemd-nspawn.xml

```diff
@@ -1017,6 +1017,26 @@
         capability names and exit.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--ambient-capability=</option></term>
+
+        <listitem><para>Specify one or more additional capabilities to
+        pass in the inheritable and ambient set to the program started
+        within the container. The value <literal>all</literal> is not
+        supported for this setting.</para>
+
+        <para>All capabilities specified here must be in the set
+        allowed with the --capability and --drop-capability
+        options. Otherwise, an error message will be shown.</para>
+
+        <para>This option cannot be combined with the boot mode of the
+        container.</para>
+
+        <para>If the special value of <literal>help</literal> is
+        passed, the program will print known capability names and
+        exit.</para></listitem>
+      </varlistentry>
```

Comment:
hmm, so far we got away with not specifying in the explanations for --capability= and --capability-drop= which caps sets these optoins affect. But if there's now control for the ambient caps, we need to be clearer for the other option too i guess. hence please update the relevant explanations about this too. this can be brief, key is just to say they affect the caps bounding set (and thus also the other two).